src/migrations/20171015155157_reports_designated.ts

```diff
@@ -0,0 +1,26 @@
+import * as Knex from "knex";
+
+exports.up = async (knex: Knex): Promise<any> => {
+  return knex.schema.dropTableIfExists("reports_designated").then( (): void  => {
```

Comment:
I believe @pgebheim's recent patch changed all names to use camel case names instead of snake case -- should probably use that convention here as well.  (Unless table names need to be exempt from that rule for some reason?)

Reply:
Also, minor style point: there is an extra space in `then( (): void  => {` between the two initial parens, and before the arrow.  As with other style gripes, I don't feel super strongly about this point, other than that I think the code base overall should have a uniform style.

Reply:
Note: style gripe occurs elsewhere as well, just bringing it up here.  Are there settings for this in tslint?  That would be ideal if so.

Reply:
table names are still under-score'd and lower (the only existing one i went off is `blockchain_sync_history`). Since they are represented as files on disk, not mixing case feels right, but i don't feel strongly. @pgebheim  ?

Reply:
re: extra space, you're right, i'm surprised linter didn't catch that? Anyway, i copied from:

https://github.com/epheph/augur-node/blob/d23310dfa7110c310e336e9d9972746682f9af67/src/migrations/20171011152617_reports.ts#L4

i'll fix there, too

Reply:
Hm, is it possible the linter isn't fully recursing?  I wonder if it's just missing some files.

Reply:
Re: file/table names, whatever you guys decide is fine with me, so long as the table names are consistent (i.e., either all camel case or all snake case, not some of each :grin: ).